### PR TITLE
chore(bitgo): add example for avaxp export import

### PR DESCRIPTION
**Pull request by shiksb**
STLX-18197

### CODE REVIEW

File: modules/bitgo/example/ts/avaxp/multisig-crosschain-export-import.ts [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+const conf = FUJI;
+
+// Private Key containing funds
+const avaxPrivateKey = 'PrivateKey-e2PZP3FYBU3G3gkRXXZkYPtye2ZqFrbSA13uDiutLK5ercixX';
+
+// multisig wallet receive addresses
+const pMultisigReceiveAddresses = [
+  'P-fuji1g9xy3vv537dncm2438t2kpk3jq3rm9sfaamcsp',
+  'P-fuji1cgygucxnhe3daq8wh7n5557qf8t6gy5ravugxs',
+  'P-fuji1d0ynjkgjaranlztj7swl04awtxeksl5wpd2puz',
```
Comment (jicli) [inline modules/bitgo/example/ts/avaxp/multisig-crosschain-export-import.ts:57]: are we interested in remove the specific value but leaving example like 
`PrivateKey-here`
`P-address1`
`P-address2`

**Review by jicli — DISMISSED** [review]

File: modules/bitgo/example/ts/avaxp/multisig-crosschain-export-import.ts [hunk 2]
```diff
@@ -0,0 +1,252 @@
+import { Avalanche, BinTools, BN, Buffer } from 'avalanche';
```
Comment (mmcshinsky-bitgo) [inline modules/bitgo/example/ts/avaxp/multisig-crosschain-export-import.ts:1]: Can we get a detailed description outlining this example at the top of this like the other examples?

**Review by mmcshinsky-bitgo — CHANGES_REQUESTED** [review]

File: modules/bitgo/example/ts/avaxp/multisig-crosschain-export-import.ts (hunk 2 above)
Comment (shiksb) [reply modules/bitgo/example/ts/avaxp/multisig-crosschain-export-import.ts:6]: okay, done

Comment (mmcshinsky-bitgo) [reply modules/bitgo/example/ts/avaxp/multisig-crosschain-export-import.ts:6]: Thanks!

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Pull request merged**